archivesunleashed/auk
# Add Zoom and fullscreen controls to sigmaJS (issue #94)

## What does this do?

- Addresses #94
- Provides zoom in and out, fullscreen controls and restore default settings.
- Adds glyphicons for future icon-related bootstrappy goodness.

## How should you test it?

It should pass Rubocop and eslint.
It should provide a full-screen modal, zoom in and out and restore/refresh the screen when you click the appropriate icons.
It should adjust the sizing when you resize the window.

## What else should you know?
This runs two instances of sigma, but both run from the same dataset.   One on a modal and the regular one.  That means resizing and playing with the fullscreen network will not change the settings on the original page.  So far I cannot think of any major ui problems with that, but it should be noted.

I have only tested on firefox and chrome.  I used bootstrap for the most part to manage things, so it *should* work on other outputs too.
Fullscreen is actually fullscreen * 0.85 to accommodate for the modal header (with the collection title).  That another ui thing to think about maybe.

![image](https://user-images.githubusercontent.com/111091/39146722-91baf780-4705-11e8-9b9d-c957d89ab337.png)

![image](https://user-images.githubusercontent.com/111091/39146750-a63ece5c-4705-11e8-8a34-e35dce09ac2a.png)
